### PR TITLE
Fix #1 Enable EditorConfig spec test through editorconfig-core-test

### LINKED ISSUE #1: Enable EditorConfig spec test through editorconfig-core-test
See https://github.com/editorconfig/editorconfig-core-test

### PR DESCRIPTION
**Pull request by ppalaga**
@angelozerr now the tough part: the spec tests. While the tests should be set up properly, the success rate is quite small:

```
21% tests passed, 144 tests failed out of 182
```

The README describes how to run them (although I have no idea whether they can be run also on Windows). I have set Travis up to run them too.

### CODE REVIEW

File: src/main/java/org/eclipse/ec4j/cli/Cli.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
+/**
+ *  Copyright (c) 2017 Angelo ZERR.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  Contributors:
+ *  Angelo Zerr <angelo.zerr@gmail.com> - initial API and implementation
```
Comment (angelozerr) [inline src/main/java/org/eclipse/ec4j/cli/Cli.java]: Please add your name

**Comment by ppalaga** [comment]
Hm... something is different on Travis because there is 0% success there: https://travis-ci.org/angelozerr/ec4j/builds/284095123#L2125 Let me fix when I find some time again.

**Comment by angelozerr** [comment]
@ppalaga I have accepted your PR because I do a little refactoring.

**Comment by ppalaga** [comment]
> I have accepted your PR because I do a little refactoring.

Yeah, we can fix those little things later.

**Comment by ppalaga** [comment]
BTW, with master on 8c9a69dfe762d7891c1c6d62742b335c9a118f24 I see `38% tests passed, 112 tests failed out of 182` so we perhaps also have some Linux vs. Windows problem.

**Pull request merged**